examples/gem/lib/gem/BUILD

```diff
@@ -32,3 +33,10 @@ rb_binary(
     args = ["lib/gem/version.rb"],
     deps = [":version"],
 )
+
+run_binary(
```

Comment:
That's the first time I run into this rule. Is this a common way to run binary targets? I felt like `bazel run` cli is the default one and nobody normally uses binary targets as dependencies to other targets.

Reply:
They are for two very different use cases. Bazel run is for the side effects of running the program, run_binary is for using the program to produce outputs into bazel-out.